xbmc/profiles/ProfilesManager.cpp

```diff
@@ -281,9 +281,10 @@ bool CProfilesManager::LoadProfile(size_t index)
 
   CServiceBroker::GetInputManager().SetMouseEnabled(CServiceBroker::GetSettings().GetBool(CSettings::SETTING_INPUT_ENABLEMOUSE));
 
-  g_infoManager.ResetCache();
-  g_infoManager.GetInfoProviders().GetGUIControlsInfoProvider().ResetContainerMovingCache();
-  g_infoManager.GetInfoProviders().GetLibraryInfoProvider().ResetLibraryBools();
+  CGUIInfoManager& infoMgr = CServiceBroker::GetGUI()->GetInfoManager();
```

Comment:
this should check for nullptr, too. iirc profiles get loaded before ui

Reply:
okay.